### DISCUSSION TITLE
Notification for failed speedtest runs

### DISCUSSION THREAD

**Discussion by armond-avanes** [alexjustesen/speedtest-tracker#1900] [Q&A]
Is any notification triggered when a speedtest run fails completely (status=fail --- no internet at all, etc) ? I need to perform some local automations in my LAN (using webhooks) if the internet is completely out.

**Comment by alexjustesen** [comment]
Not currently but it's planned, run me though a use case because I've avoided failed speedtest notifications so far because they nearly all require a WAN connection.

**Reply by armond-avanes** [reply]
Thanks for your response. My guess is that it's going to be after 1.0.0... correct?

As for my use case, recently I've been having some fluctuations with my internet connection and sometimes it completely drops. The only solution that I have is to turn the modem off and back on to make it work again! That's what I'm trying to automate.

While we're on notification topic, it will be great if you can document the payload/json structure when you trigger a [webhook] notification for completed, failed threshold and later complete failures. I couldn't find it anywhere in the documentations.

**Reply by svenvg93** [reply]
> While we're on notification topic, it will be great if you can document the payload/json structure when you trigger a [webhook] notification for completed, failed threshold and later complete failures. I couldn't find it anywhere in the documentations.

I listed it in https://github.com/alexjustesen/speedtest-tracker-docs/issues/49 for the V1 docs update :)

**Reply by alexjustesen** [reply]
> As for my use case, recently I've been having some fluctuations with my internet connection and sometimes it completely drops. The only solution that I have is to turn the modem off and back on to make it work again! That's what I'm trying to automate.

I got ya, and that sounds like a good automation opportunity! So a webhook for a failure would be enough?

**Reply by armond-avanes** [reply]
> I listed it in [alexjustesen/speedtest-tracker-docs#49](https://github.com/alexjustesen/speedtest-tracker-docs/issues/49) for the V1 docs update :)

Sounds great! Thank you!

**Reply by armond-avanes** [reply]
> I got ya, and that sounds like a good automation opportunity! So a webhook for a failure would be enough?

Yes, for my use case that would be enough.
